### PR TITLE
refactor: Remove group toast tip

### PR DESCRIPTION
**Pull request by MoonLeeeaf**
# 重构 移除修改消息设置提示

## Description

添加版本判断，移除了重复的文件

#1112 的延续

## Check List

- [x] I have tested the changes and verified that they work and don't break anything(as well as I can manage) or drop the support for previous versions.
- [x] I have merged commits that are meaningless for follow-up work and confirmed that they will not cause damage to follow-up maintenance

本人不熟悉Kotlin，可能会有非常非常非常非常非常非常多次对于错误语法的修补提交，或考虑合并（摊

下次得用Java写了（（（（（（（（

### CODE REVIEW

File: app/src/main/java/cc/microblock/hook/TimRemoveToastTips.kt [hunk 1]
```diff
@@ -32,30 +32,45 @@ import io.github.qauxv.hook.CommonSwitchFunctionHook
...
-            it.result = null;
-        }
-        // 漏了一个“你可以在这里xxxxx”，实在吐了，这玩意不定时的
-        HookUtils.hookBeforeIfEnabled(
-            this, Reflex.findMethod(
-                Initiator.loadClass("com.tencent.mobileqq.activity.aio.rebuild.TroopChatPie\$38"),
-                "run")
-        ) {
-            it.result = null;
+
```
Comment (cinit) [inline app/src/main/java/cc/microblock/hook/TimRemoveToastTips.kt:47]: 把这个加上去方便以后参考
```suggestion
// 功能基于 Issue #781 和 #667 移植实现
```

**Comment by cinit** [comment]
2f947ab0f9034e8df053d834420a56903652c00f

**Comment by cinit** [comment]
辛苦叶叶了

**Comment by MoonLeeeaf** [comment]
恕我对Kotlin了解不足，辛苦您了

**Pull request closed without merging**